src/github/middleware.ts

```diff
@@ -123,7 +133,7 @@ export default (
 				gitHubInstallationId.toString()
 			);
 			context.sentry.setUser({ jiraHost, gitHubInstallationId });
-			context.log = logger.child({ jiraHost });
+			context.log = context.log.child({ jiraHost });
```

Comment:
The reason I used an external const of `logger` is because all of these function will reference `context.log` then create a child of each other instead of all having the same parent.